src/components/ui/PopOver/Styled.tsx

```diff
@@ -6,7 +6,7 @@ import { ellipsis } from '../../../utils/style';
 
 export const StyledPopOverMenu = styled.ul`
   width: fit-content;
-  max-width: 22rem;
+  max-width: 33rem;
```

Comment:
If you need to change the styling for a specific implementation. You are able to pass any styles through the `css` prop or other styling methods.

Reply:
Will add a classname to StyledPopOverMenu. Then overwrite the css style by this.